src/debugger.rs

```diff
@@ -64,6 +65,7 @@ pub fn start_debugger<P: AsRef<Path>>(
     .host_network(true)
     .name(DEBUG_SERVER_NAME.to_string())
     .daemon(tty)
+    .tty(!tty)
```

Comment:
tty default is `false`, this line is not necessary.

Reply:
well, you're right, it is not necessary. :)